charm/config.yaml

```diff
@@ -7,3 +7,7 @@ options:
     default: 10
     description: Number of seconds to wait for keepalive connections
     type: int
+  api_port:
```

Comment:
Remove this section, we don't need api_port in the config, and it was actually recommended that we do not do this. Hardcoding things like this may seem unusual, but there's really no reason for it to change, and removing the option reduces the chances for config mismatches.

Reply:
No problem